### ISSUE TITLE
Compression option is not supported.

### ISSUE THREAD

**Issue by slavitch** [gravitational/teleport#13534]
tsh ssh -o 'Compression yes' -o 'CompressionLevel 6' c2ro@hostname
WARNING: Option 'Compression' is not supported.
WARNING: Option 'CompressionLevel' is not supported.

**Comment by zmb3** [comment]
Right. `crypto/ssh` does not support compression, but I've tagged this with `feature-request` to reflect the ask.
